recipes/pyexcel-ods3/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyexcel-ods3" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "0163ea8d6c2d3bfdb52cf1c0c5fcc2fb9dce467547077558f7cd9df7c2797d57" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
Are the other packages beyond `python` and `setuptools` required to build this package? With `--single-version-externally-managed` you typically only need what `setup.py` imports (directly or indirectly), unless there is a `setup_requires` section.